src/frontend/package.json

```diff
@@ -31,8 +27,8 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "start": "npm run develop",
     "serve": "gatsby serve",
+<<<<<<< HEAD
```

Comment:
You have failed merge conflict markers in here causing the JSON parser to fail.

Reply:
Yeah, I see. Fixing it now

Reply:
@humphd I have updated my files but now I'm getting new error
![image](https://user-images.githubusercontent.com/49209351/74058976-252f8180-49b5-11ea-825d-39ba78d98796.png)
I don't know what to do to fix this error